scripts/analysis.py

```diff
@@ -690,28 +756,26 @@ def where_comm(cursor, commodity, prototypes):
     commodity: str
         name of commodity
     prototypes: list
-        list of prototypes from
+        list of prototypes that provides commodity
 
     Returns
     -------
-    Dictionary of timeseries of commodity sent from prototypes
+    trade_dict: dictioary
+        dictionary with key: prototype name,
+        value: timeseries of commodity sent from prototypes
```

Comment:
I think this colon is going to confuse sphinx. Please re-write these key value parts of your docs with equal signs throughout this pull request.